pjesak.pde

```diff
@@ -21,41 +30,51 @@ class Pjesak implements Collideable{
     y1 = (float) (level.centerY(tmpY) - h / 2);
     x2 = (float) (level.centerX(tmpX + tmpW - 1) - w / 2);
     y2 = (float) (level.centerY(tmpY + tmpH - 1) - h / 2);
-    distance = dist(x1, y1, x2, y2); // duljina pjesakovog puta
+    distance = dist(x1, y1, x2, y2); // udaljenost od (x1, y1) do (x2,y2)
 
     dir = getDirection(attrib.get("direction"));
-    if(dir==Direction.DOWN) path = 1;
-    else if (dir==Direction.UP) path = 0;
+    if(dir == Direction.DOWN) path = 1;
+    else if (dir == Direction.UP) path = 0;
     else if (dir == Direction.LEFT) path = 1;
     else path = 0;
+    
+    // Odredi poziciju pjesaka - odaberi stranu ceste s koje krece?

```

Comment:
Prije se odredilo na kojoj strani kreće pješak, sad se samo izračuna koje su to koordinate.